core/src/main/kotlin/links/DRI.kt

```diff
@@ -23,6 +23,10 @@ data class DRI(
     }
 }
 
+enum class DRIExtra(val value: String) {
```

Comment:
I am not a fan of this design. It has a fundamental flaw of not being extensible to plugin creators. Imho a good extra should be extensible, type safe and (possibly) easily serialisable